src/lib/web-worker/worker-exec.ts

```diff
@@ -40,10 +41,12 @@ export const initNextScriptsInWebWorker = async (initScript: InitializeScriptDat
 
       rsp = await fetch(scriptSrc);
       if (rsp.ok) {
-        scriptContent = await rsp.text();
-
-        env.$currentScriptId$ = instanceId;
-        run(env, scriptContent, scriptOrgSrc || scriptSrc);
+        let responseContentType = rsp.headers.get("content-type")
+        if (!contentTypeBlocklist.some(ct => ct === responseContentType)){
```

Comment:
This is a good idea, but rather should go with a whitelist instead of blacklist.

Reply:
Done